appveyor.yml

```diff
@@ -33,3 +33,19 @@ artifacts:
     name: Daedalus Win64 Installer
   - path: node_modules\daedalus-client-api\build-id
     name: cardano-sl-build-id
+
+deploy:
+  provider: S3
+  access_key_id:
+    secure: gExbu9JE+yzLdqhjKeZDC9kP8XyFhdVfo2Uaacnhe04=
+  secret_access_key:
+    secure: ILqyRuq76HAze5Yg+aD5aiyxgV4jSRtbMopV0GDMAx+2AveakN2LZmheCQmRPFuU
```

Comment:
won't anyone with access to these access key and secret access key be able to upload installers to the S3 buckets?

Reply:
The values here are encrypted with Appveyor [secure variables](https://www.appveyor.com/docs/build-configuration/#secure-variables).

Reply:
Awesome, thanks for the clarification. The access key sure looked different than usual :)